packages/ui/src/Card.module.scss

```diff
@@ -11,25 +11,21 @@
 
     display: flex;
     align-items: center;
-    margin-bottom: c.$grid * 4;
 
     .icon {
       margin-right: c.$grid * 2.5;
     }
 
     .title {
       margin: 0;
-
-      &.space {
-        margin-right: c.$grid * 2.5;
-      }
+      margin-right: c.$grid * 2.5;
```

Comment:
I wouldn't change this. Extra 10px on a small screen is priceless. What do you thinks @utkukaratas @fantkolja  @allfayn?

Reply:
@Sodik You're right, I wanted to revert this. Here it is: https://github.com/hazelcast/hive/pull/347/commits/b107f6a6cee6705a5dbe0e950e82aecc7c7c7ba0